client/homebrew/brewRenderer/tutorialPopup/tutorial.jsx

```diff
@@ -0,0 +1,161 @@
+const React = require('react');
+const createClass = require('create-react-class');
+const _ = require('lodash');
+require('./tutorial.less');
+
+const DISMISS_KEY = 'dismiss_tutorial';
+
+const TutorialPopup = createClass({
+    displayName: 'TutorialPopup',
+
+    getInitialState: function () {
+        return {
+            steps: {},
+            currentStep: 'splitPaneStep', // Initialize to the first step
+        };
+    },
+
+    componentDidMount: function () {
+        this.checkTutorial();
+        window.addEventListener('resize', this.checkTutorial);
+    },
+
+    componentWillUnmount: function () {
+        window.removeEventListener('resize', this.checkTutorial);
+    },
+
+    steps: {
+        splitPaneStep: function () {
+            window.localStorage.setItem(
+                'naturalcrit-pane-split',
+                window.innerWidth / 2
+            );
+            return (
+                <div className="content">
+                    <div className="editor">
+                        <span className="explanation editor">
```

Comment:
You could just use `<p>` tags instead of spans, and then style them how you want.

Reply:
`p` elements are block level elements to describe paragraphs. `spans` are used to apply CSS style and/or class(es) to an arbitrary section of text and inline elements. I believe my choice to be more appropiate for this specific use.